subsys/bluetooth/shell/mcc.c

```diff
@@ -1348,12 +1441,21 @@ SHELL_STATIC_SUBCMD_SET_CREATE(mcc_cmds,
 	SHELL_CMD_ARG(read_current_track_obj_id, NULL,
 		      "Read Current Track Object ID",
 		      cmd_mcc_read_current_track_obj_id, 1, 0),
+	SHELL_CMD_ARG(set_current_track_obj_id, NULL,
+		      "Set Current Track Object ID",
+		      cmd_mcc_set_current_track_obj_id, 2, 0),
 	SHELL_CMD_ARG(read_next_track_obj_id, NULL,
 		      "Read Next Track Object ID",
 		      cmd_mcc_read_next_track_obj_id, 1, 0),
+	SHELL_CMD_ARG(set_next_track_obj_id, NULL,
+		      "Set Next Track Object ID",
+		      cmd_mcc_set_next_track_obj_id, 2, 0),
 	SHELL_CMD_ARG(read_current_group_obj_id, NULL,
 		      "Read Current Group Object ID",
 		      cmd_mcc_read_current_group_obj_id, 1, 0),
+	SHELL_CMD_ARG(set_current_group_obj_id, NULL,
+		      "Set Current Group Object ID",
+		      cmd_mcc_set_current_group_obj_id, 2, 0),
```

Comment:
Considering adding some way of showing that it expects an ID argument in base 10

Reply:
It doesn't.  Seems to work perfectly well with both hex, octal and decimal input.

Reply:
Still missing some information about what the argument is expected to be. See e.g. the `bt.c` shell commands for how arguments are defined in the help string.

Reply:
Got it.  Fixed.